### PR TITLE
use dedicated istio service account

### PR DESCRIPTION
**Pull request by krzyzacy**
I think we have to use the service account within the same project to have alpha api access

/assign @ostromart @yliaog

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -15380,7 +15393,7 @@ periodics:
   agent: kubernetes
   name: istio-periodic-e2e-gke-addon
   labels:
-    preset-service-account: true
+    preset-istio-service: true
```
Comment (BenTheElder) [inline prow/config.yaml]: can this do uploads to the GCS bucket?
/shrug

Comment (BenTheElder) [reply prow/config.yaml]: /hold

Comment (krzyzacy) [reply prow/config.yaml]: I'll whitelist that

Comment (krzyzacy) [reply prow/config.yaml]: the job is busted anyway

**Pull request merged**